### PR TITLE
KEYCLOAK-19096 Add default labels to CSV Realm CR example

### PR DESCRIPTION
**Pull request by jonathanvila**
In order to test this PR :

1. create an Openshift 4.7 instance using the Katacoda [playground](https://www.katacoda.com/courses/openshift/playgrounds/openshift47)
1. create a catalog-source.yaml
```bash
apiVersion: operators.coreos.com/v1alpha1
kind: CatalogSource
metadata:
  name: test-catalog
  namespace: openshift-marketplace
  labels:
    application: keycloak
spec:
  sourceType: grpc
  image: quay.io/{your account}/keycloak-operator-test-catalog:{version}
   ```
3. remove the replace line (last one) in the CSV file to allow creating a catalog with only this version
3. create a test catalog bundle to install this test operator
```bash
docker build -f bundle.Dockerfile -t keycloak-operator-bundle:100 .

docker tag keycloak-operator-bundle:100 quay.io/{your account}/keycloak-operator-bundle:100

docker push quay.io/{your account}/keycloak-operator-bundle:100

~/{your path to operator-registry project}/operator-registry/bin/opm index add --bundles quay.io/{your account}/keycloak-operator-bundle:100 \
--tag quay.io/{your account}/keycloak-operator-test-catalog:100 --container-tool docker

docker push quay.io/{your account}/keycloak-operator-test-catalog:100

sed "s/{version}/100/g" catalog-source.yaml | kubectl apply -f -
```
4. install the operator from the OperatorHub screen in Openshift using the one on the test-catalog
1. create the keycloak, realm and client CRs with the default information
1. log to the keycloak admin and check that for the realm created there's the client created

### CODE REVIEW

File: deploy/olm-catalog/keycloak-operator/15.0.1/manifests/keycloak-operator.v15.0.1.clusterserviceversion.yaml [hunk 1]
```diff
@@ -35,7 +35,10 @@ metadata:
           "apiVersion": "keycloak.org/v1alpha1",
           "kind": "KeycloakRealm",
           "metadata": {
-            "name": "example-keycloakrealm"
+            "name": "example-keycloakrealm",
+            "labels": {
+              "app": "sso"
+            }
```
Comment (vmuzikar) [inline deploy/olm-catalog/keycloak-operator/15.0.1/manifests/keycloak-operator.v15.0.1.clusterserviceversion.yaml:41]: @jonathanvila As we discussed privately, we'll do this as part of the 15.0.2 release.

**Review by vmuzikar — CHANGES_REQUESTED** [review]
Thanks for the PR! The readme change LGTM.

File: deploy/olm-catalog/keycloak-operator/15.0.1/manifests/keycloak-operator.v15.0.1.clusterserviceversion.yaml (hunk 1 above)
Comment (jonathanvila) [reply deploy/olm-catalog/keycloak-operator/15.0.1/manifests/keycloak-operator.v15.0.1.clusterserviceversion.yaml:41]: :+1:

**Comment by jonathanvila** [comment]
Incorporated by : https://github.com/keycloak/keycloak-operator/pull/375

**Pull request closed without merging**